.github/workflows/sanity.yml

```diff
@@ -78,6 +78,7 @@ jobs:
               "python-version": "3.9"
             }
           ]
+      collection_pre_install: '-r source/tests/sanity/requirements.yml'
```

Comment:
I'm having trouble understanding the reason behind this action in the sanity workflow. Could you please clarify why this is being done and whether it addresses any specific issues or failures?

Reply:
Removed, I needed them for a previous test, but they are not needed anymore! thanks!